scripts/h2/schema-0.7.0-h2.sql

```diff
@@ -93,6 +93,7 @@ CREATE TABLE IF NOT EXISTS `table_column_version_info` (
     `table_version` INT UNSIGNED NOT NULL COMMENT 'table version',
     `column_id` BIGINT(20) UNSIGNED NOT NULL COMMENT 'column id',
     `column_name` VARCHAR(128) NOT NULL COMMENT 'column name',
+    `column_position` INT UNSIGNED NOT NULL COMMENT 'column position',
```

Comment:
Is the value start from `0` or `1`? Please add this description here.

Reply:
Starting from 0, I will add some comments.